exporters/tests/test_in_memory_metric.py

```diff
@@ -61,7 +64,7 @@ def teardown_method(self):
         [
             (
                 "todolist",
-                "1678269658",
+                str(CURRENT_TIMESTAMP),
```

Comment:
No need to change here, its commit metric, not deploy

Reply:
Similar comment to the one below:

```
actually it will be valid to only accept timestamps from the time of starting the application. I know that at the moment it's not really valid, but really it does not change the test logic, just modifying the time to be current and not in the past, so I think it's not harmful change here.
```